src/System.IO.FileSystem.Primitives/tests/System.IO.FileSystem.Primitives.Tests.csproj

```diff
@@ -23,7 +23,10 @@
     <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\System.IO.FileSystem.Primitives\pkg\System.IO.FileSystem.Primitives.pkgproj">
+    <ProjectReference Include="..\..\System.Runtime\pkg\System.Runtime.pkgproj">
```

Comment:
Small Nit: pkgproj references don't need metadata, so you can remove all that, plus you should add a ToDo comment to remove System.Runtime reference